azext_iot/operations/dps.py

```diff
@@ -156,7 +163,7 @@ def iot_dps_device_enrollment_create(
 
         if attestation_type == AttestationType.tpm.value:
             if not endorsement_key:
-                raise CLIError("Endorsement key is requried")
+                raise RequiredArgumentMissingError("Endorsement key is requried")
```

Comment:
need to include the parameter needed --endorsement-key

Reply:
Also typo 'requried'  ->  'required'